suite-native/module-home/src/components/PortfolioGraph.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Box, Text } from '@suite-native/atoms';
+
+export const PortfolioGraph = () => (
+    <Box>
+        <Text variant="titleSmall">$3,720</Text>
```

Comment:
I guess it shouldn't be `titleSmall`? Also, why not add the green and red arrows next to the percentage right away? ;)

![image](https://user-images.githubusercontent.com/45338719/180267221-0bab1a7a-0ca2-40de-98c6-d205232e8455.png)

Reply:
Title small is correct, as it is in Figma :) @dahaca

Reply:
Perhaps I was looking at at the wrong version? Could you share the one you are working with?

Reply:
Wrong file for me. It is actually titleLarge as discussed.